app/src/main/java/wannabit/io/cosmostaion/activities/HtlcResultActivity.java

```diff
@@ -522,8 +522,8 @@ public void run() {
     //SWAP ID LOOP CHECK
     private void onShowMoreSwapWait() {
         AlertDialogUtils.showDoubleButtonDialog(this, getString(R.string.str_more_wait_swap_title), getString(R.string.str_more_wait_swap_msg),
-                getString(R.string.str_close), view -> onBackPressed(),
-                getString(R.string.str_wait), null, false);
+                getString(R.string.str_close), view -> onFinishWithError(),
```

Comment:
onFinishWithError right?

Reply:
yes sir! it's my mistake. And the progress bar was also refactored. Please check and give feedback anytime. Thank you!